ldapsync/ldapsyncapp.py

```diff
@@ -55,7 +89,34 @@ def __init__(self):
             file_handler.setFormatter(formatter)
             self.logger.addHandler(file_handler)
 
-
+    @property
     @abc.abstractmethod
-    def sync(self):
+    def dest_service(self):
+        """Return the destination service to sync to."""
         pass
+
+    def sync(self):
+        try:
+            for ldap_group, dest_group in self.SYNC_PAIRS:
```

Comment:
This is fine for now, but if we're going to separate each service into a different "run", it could be worth separating even further and running at the group level. Though also it depends on exactly how we end up scheduling these 
¯\\_(ツ)_/¯

Reply:
This is just a stopgap for now; I'd like to write a YAML config file for each sync pair--I was thinking something like:

```
-ldap_group: 
  -dest_service: ['permission_1', 'permission_2']
  ...
...
```

Would it be preferable to pass each sync pair in as arguments? I could do that too.